src/main/java/mate/jdbc/service/DriverService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
```

Comment:
```suggestion
   Driver get(Long id);
```
<img width="759" alt="image" src="https://user-images.githubusercontent.com/81781479/228931757-b90cce5f-a91b-49bf-a382-b6e25847de88.png">